lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -206,39 +204,63 @@ public function invalidateToken( $token, $userId )
 	 *
 	 * @return array - JSON string deserialized into an associative array
 	 */
-	public function generateToken( $userId )
-	{
+	public function generateToken( $userId ) {
 		return $this->request(
-			sprintf('users/%s/tokens', $userId),
+			sprintf( 'users/%s/tokens', $userId ),
 			[],
 			[],
 			[ 'method' => 'POST' ]
 		);
 	}
 
 	/**
-	* A shortcut method for register requests.
-	*/
-	public function register( $username, $password, $email, $birthdate, $langCode )
-	{
-			// Convert the array to URL-encoded query string, so the Content-Type
-			// for the POST request is application/x-www-form-urlencoded.
-			// It would be multipart/form-data which is not supported
-			// by the Helios service.
-			$postData = http_build_query( [
-				'username'  => $username,
-				'password'  => $password,
-				'email'     => $email,
-				'birthdate' => $birthdate,
-				'langCode'  => $langCode,
-			] );
-
-			return $this->request(
-				'users',
-				[],
-				$postData,
-				[ 'method'	=> 'POST' ]
-			);
+	 * A shortcut method for register requests.
+	 */
+	public function register( $username, $password, $email, $birthdate, $langCode ) {
+		// Convert the array to URL-encoded query string, so the Content-Type
+		// for the POST request is application/x-www-form-urlencoded.
+		// It would be multipart/form-data which is not supported
+		// by the Helios service.
+		$postData = http_build_query( [
```

Comment:
What do you think of making the param passed to request and array and move the `http_build_query` call there as we do for get data, rather than having each method do it separately?